Automattic/jetpack
# Enhance/language generation

Replaces #4975 for now.

#### Changes proposed in this Pull Request:
-  Makes sure that we only ship the strings we need in JSON files rather than everything we have.
-  Reduces the JSON file size approximately by 90%. 

### To test
If you don't want to wait for languages to load, just remove the dependency in gulpfile on line 480~ something. You need to remove `[ 'languages:get' ]` from the task description, and the script will use your existing .po files to build translations. Keep in mind that we only have some .po files, so some of the JSON files will not be regenerated this way.
In order to test you can drop all *.json files in `languages/json`. After that you need to make sure that JSON includes only the strings that are defined in `_inc/jetpack-strings.php`.
